fern/css/software-stack-diagram.css

```
@@ -0,0 +1,547 @@
+    left: 50%;
+    transform: translateX(-50%);
+    padding: 8px 12px;
+    border-radius: 6px;
+    background: var(--pst-color-surface);
+    border: 1px solid var(--nv-color-green);
+    color: var(--pst-color-text-base);
+    font-size: 12px;
+    max-width: 500px;
+    width: max-content;
+    white-space: nowrap;
+    word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)